### PR TITLE
mes 6085 remove safety competency from test report

### PR DESCRIPTION
**Pull request by piercemorris**
## Description

Remove the safety competency button from the test report page for category D, D1 and DE, D1E.

**NB** Files and folders were renamed from references of VehicleChecksCompactCatDComponent to VehicleChecksComponent to allow for consistency that is seen throughout the other category pages providing vehicle check components - since the button is no longer _compact_ but stretched to will width of row.

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

### Category D, D1
Simulator Screen Shot - iPad Pro (10 5-inch) - 2020-11-19 at 14 44 21

### Category DE, D1E
Simulator Screen Shot - iPad Pro (10 5-inch) - 2020-11-19 at 15 06 03

### CODE REVIEW

**Review by ammarhaiderbjss — APPROVED** [review]
Looks good 😍  I like the renaming

File: src/pages/test-report/cat-d/components/test-report.cat-d.components.module.ts [hunk 1]
```diff
@@ -4,13 +4,13 @@ import { IonicModule } from 'ionic-angular';
...
-import { VehicleChecksCompactCatDComponent } from './vehicle-checks-compact/vehicle-checks-compact.cat-d';
+import { VehicleChecksComponent } from './vehicle-checks/vehicle-checks';
 import { SafetyQuestionsCatDComponent } from './safety-questions/safety-questions.cat-d';
 import { PcvDoorExerciseComponent } from './pcv-door-exercise/pcv-door-exercise';
 
 @NgModule({
   declarations: [
-    VehicleChecksCompactCatDComponent,
+    VehicleChecksComponent,
     SafetyQuestionsCatDComponent,
```
Comment (szabikr) [inline src/pages/test-report/cat-d/components/test-report.cat-d.components.module.ts]: The `SafetyQuestionsCatDComponent` is not used anymore so would be a good idea to remove it from the declarations. And actually, the entire component can be deleted

Comment (piercemorris) [reply src/pages/test-report/cat-d/components/test-report.cat-d.components.module.ts]: Ok I didn't know about this, thanks

Comment (piercemorris) [reply src/pages/test-report/cat-d/components/test-report.cat-d.components.module.ts]: @szabikr the component has been removed

**Review by szabikr — APPROVED** [review]
awesome! Looks good

**Pull request merged**